### ISSUE TITLE
Logo inside QR code format support.

### ISSUE THREAD

**Issue by silverSY** [zxing/zxing#1305]
can you add support for this QR code
![qr](https://user-images.githubusercontent.com/38359875/90672421-f4a18d80-e20a-11ea-88d5-c978693331a3.png)

**Comment by srowen** [comment]
Yeah that QR code appears to be invalid - not sure it's the logo but something about how it was encoded.
